packages/subgraph/src/mappings/cfav1.ts

```diff
@@ -202,11 +203,20 @@ export function handleFlowOperatorUpdated(event: FlowOperatorUpdated): void {
         event.params.token,
         event.params.sender
     );
-
     flowOperator.permissions = event.params.permissions;
     flowOperator.flowRateAllowanceGranted = event.params.flowRateAllowance;
     flowOperator.flowRateAllowanceRemaining = event.params.flowRateAllowance;
     flowOperator.flowOperator = event.params.flowOperator;
+
```

Comment:
Not sure, do we need to call (RPC) allowance here or not.

Reply:
Moved into getOrInitFlowOperator, it will only be called when the entry does not exist. Previously, it was being called every time the user updated their permission or flow allowance.

Reply:
nice!